Scripts/assignment/assignment_period.py

```diff
@@ -483,20 +483,21 @@ def _calc_background_traffic(self, include_trucks=False):
         # calc @bus and data3
         heavy = (self.extra("truck"), self.extra("trailer_truck"))
         for link in network.links():
-            freq = 0
-            for segment in link.segments():
-                segment_hdw = segment.line[self.extra("hw")]
-                if 0 < segment_hdw < 900:
-                    freq += 60 / segment_hdw
-            link[self.extra("bus")] = freq
-            if link.type // 100 in param.bus_lane_link_codes[self.name]:
-                # Bus lane
-                link[background_traffic] = 0
-            else:
-                link[background_traffic] = freq
-            if include_trucks:
-                for ass_class in heavy:
-                    link[background_traffic] += link[ass_class]
+            if link.type > 100: # If car or bus link
```

Comment:
Will link types 0-100 now be considered as vdf=0?

Reply:
No, not because of this code. This code forces attribute `@bus` to zero for rail links.